.swiftlint.yml

```diff
@@ -11,20 +11,29 @@ disabled_rules:
     - for_where #使用 `for where` 代替 简单的 `for { if }`
     - class_delegate_protocol #delegate protocol 应该被设定为 class-only,才能被弱引用
     - todo #避免 TODOs and FIXMEs 标识
+    - line_length
+    - function_body_length
+    - no_space_in_method_call
+    - multiple_closures_with_trailing_closure
+    - switch_case_alignment
+    - syntactic_sugar
```

Comment:
I don't think disable those rules are good practices

Reply:
> I don't think disable those rules are good practices

The first 5 cause compilation to fail in the current code, so keeping them means refactoring.

- `no_space_in_method_call` causes warnings reported on cases where a function takes a closure and it's called without parenthesis, for example: `DispatchQueue.main.async {`
- `line_length` I personally tend to ignore it, as in some cases I prefer a longer line to splits
- `switch_case_alignment` seems to randomly throw warnings even if cases are aligned correctly
- `syntactic_sugar`: I find this annoying, as sometimes I prefer to make the code more explicit by avoiding syntactic sugar — typically in some usages of optionals, or when defining extensions.

Anyway it seems nobody but me has the linter enabled, as it causes compilation to fail.